params/protocol_params.go

```diff
@@ -60,23 +60,38 @@ const (
 	SstoreResetGasEIP2200             uint64 = 5000  // Once per SSTORE operation from clean non-zero to something else
 	SstoreClearsScheduleRefundEIP2200 uint64 = 15000 // Once per SSTORE operation for clearing an originally existing storage slot
 
+	// ColdAccountAccessCostEIP2929 (2600 -> 900), ColdSloadCostEIP2929 (2100 -> 800) are modified according to CIP-0048
+	// Links: https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0048.md
+	ColdAccountAccessCostEIP2929 = uint64(900) // COLD_ACCOUNT_ACCESS_COST
+	ColdSloadCostEIP2929         = uint64(800) // COLD_SLOAD_COST
+	WarmStorageReadCostEIP2929   = uint64(100) // WARM_STORAGE_READ_COST
+
+	// In EIP-2200: SstoreResetGas was 5000.
+	// In EIP-2929: SstoreResetGas was changed to '5000 - COLD_SLOAD_COST'.
+	// In EIP-3529: SSTORE_CLEARS_SCHEDULE is defined as SSTORE_RESET_GAS + ACCESS_LIST_STORAGE_KEY_COST
+	// Which becomes: 5000 - 2100 + 1900 = 4800
+	SstoreClearsScheduleRefundEIP3529 uint64 = SstoreResetGasEIP2200 - ColdSloadCostEIP2929 + TxAccessListStorageKeyGas
```

Comment:
Just pointing something that I've noticed, and I don't really know if it is or not an issue.
As we kept the `ColdSloadCostEIP2929` in 800, the `SstoreClearsScheduleRefundEIP3529` will be 6100
`SstoreClearsScheduleRefundEIP3529 uint64 = SstoreResetGasEIP2200(5000) - ColdSloadCostEIP2929(800) + TxAccessListStorageKeyGas(2100)`
Don't really know if that means that we are going to refund more gas than the one spent by the user.
(of course we should change the comments too)